### PR TITLE
88 add more plots to the sputtering class and create and populate more overview quantities

### PR DESCRIPTION
**Pull request by eugenebertin**
solved all issues and tested it with the--preview-plots (it works)

### CODE REVIEW

File: src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py [hunk 1]
```diff
@@ -1213,24 +1500,42 @@ class DTUSputtering(SputterDeposition, PlotSection, Schema):
...
+        #     scrollZoom=False,
+        # )
+
+        # # Adding the plotly figure to the figures list
+        # self.figures.append(
+        #     PlotlyFigure(
+        #         label='Process timeline',
+        #         figure=timeline_json,
+        #     )
+        # )
```
Comment (hampusnasstrom) [inline src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py:1538]: Does this commented out code need to stay?

File: src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py [hunk 2]
```diff
@@ -1195,6 +1440,48 @@ class DTUSputtering(SputterDeposition, PlotSection, Schema):
         type=str,
         a_eln={'component': 'RichTextEditQuantity', 'label': 'Log file report'},
     )
+    platen_used = Quantity(
+        type=MEnum(['A', 'B']),
+        a_eln={'component': 'RadioEnumEditQuantity'},
+    )
+    base_pressure = Quantity(
```
Comment (hampusnasstrom) [inline src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py:1447]: Is this something you fill manually?

File: src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py [hunk 3]
```diff
@@ -1092,28 +1184,34 @@ class DepositionParameters(ArchiveSection):
...
+    #         'label': 'Image of target before the deposition',
+    #     },
+    # )
+    # target_image_after = Quantity(
+    #     type=str,
+    #     a_eln={
+    #         'component': 'FileEditQuantity',
+    #         'label': 'Image of target after the deposition',
+    #     },
+    # ) #DISPLACED TO THE TOP LEVEL SECTION (DTUSputtering)
```
Comment (hampusnasstrom) [inline src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py:1213]: We have git for version control so there is no need to "save" code by commenting it out.

File: src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py [hunk 4]
```diff
@@ -959,10 +952,87 @@ class SourceOverview(ArchiveSection):
...
+    # min_voltage = Quantity(
+    #     type=np.float64,
+    #     a_eln={'component': 'NumberEditQuantity', 'defaultDisplayUnit': 'V'},
+    #     unit='V',
+    # )
+    # range_voltage = Quantity(
+    #     type=np.float64,
+    #     a_eln={'component': 'NumberEditQuantity', 'defaultDisplayUnit': 'V'},
+    #     unit='V',
+    # )
```
Comment (hampusnasstrom) [inline src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py:989]: See other comments

File: src/nomad_dtu_nanolab_plugin/sputter_log_reader.py
Comment (hampusnasstrom) [inline src/nomad_dtu_nanolab_plugin/sputter_log_reader.py:1]: I won't check this file in detail. As long as it works I'm happy. Let me know if you want me to review some specific part of it.

**Review by hampusnasstrom — APPROVED** [review]
Just some small comments but it looks good.

File: src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py (hunk 1 above)
Comment (eugenebertin) [reply src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py:1538]: no, it does not, I can removed it

File: src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py [hunk 5]
```diff
@@ -1213,24 +1500,42 @@ class DTUSputtering(SputterDeposition, PlotSection, Schema):
     deposition_parameters = SubSection(
         section_def=DepositionParameters,
     )
+    temp_ramp_up = SubSection(
+        section_def=TempRampUp,
+    )
+    temp_ramp_down = SubSection(
+        section_def=TempRampDown,
+    )
```
[the remark this answers is not in the document]
Comment (eugenebertin) [reply src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py:1508]: I will replace temp by temperature

File: src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py (hunk 2 above)
Comment (eugenebertin) [reply src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py:1447]: yes sometimes, I believe it makes sense to have it at the top level

File: src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py (hunk 3 above)
Comment (eugenebertin) [reply src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py:1213]: it makes sense

File: src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py (hunk 4 above)
Comment (eugenebertin) [reply src/nomad_dtu_nanolab_plugin/schema_packages/sputtering.py:989]: on the other hand, i did comment this out as I am unsure if it should be there or not. Maybe we keep it there while We decide how to structure the overview

File: src/nomad_dtu_nanolab_plugin/sputter_log_reader.py
Comment (eugenebertin) [reply src/nomad_dtu_nanolab_plugin/sputter_log_reader.py:1]: thank you I went ahead and applied your comments to a new commit

**Pull request merged**